<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winQuickLook</class>
 <widget class="QDialog" name="winQuickLook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save Quick Look</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/StarPlus.png</normaloff>icons/StarPlus.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 57 12pt &quot;TerminessTTF Nerd Font&quot;;</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Quick Look Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>95</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>95</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qleQuickLookName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
